quill-cassandra/src/main/scala/io/getquill/sources/cassandra/CassandraSourceMacro.scala

```diff
@@ -19,10 +19,10 @@ class CassandraSourceMacro(val c: Context) extends SourceMacro {
   override protected def prepare(ast: Ast, params: List[Ident]) =
     if (!IsDynamic(ast)) {
       implicit val n = LoadNaming.static(c)(namingType)
-      val (cql, idents) = Prepare(ast, params)
+      val (cql, idents, _) = Prepare(ast, params)
```

Comment:
Cassandra doesn't have the ability to add a generated value on insert and return it so either it should be ignored, or the CQL Idiom should produce an error at compile time if used.